wsrep_api.h

```diff
@@ -720,6 +719,18 @@ struct wsrep {
     wsrep_status_t (*recv)(wsrep_t* wsrep, void* recv_ctx);
 
   /*!
+   * @brief Tells provider that a given writeset has a read view associated
+   *        with it.
+   *
+   * @param wsrep  provider handle
+   * @param handle writeet handle
+   * @param rv     read view astablished by the caller or if NULL, by provider
```

Comment:
And I don't understand the description. How does this function call associates a read view with some given write set?